packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelSetupCard.tsx

```diff
@@ -1,44 +1,57 @@
-import React, { forwardRef } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import { Card, P, variables } from '@trezor/components';
 import { Translation } from '@suite-components';
+import { useSelector } from '@suite-hooks/useSelector';
+import { selectCurrentTargetAnonymity } from '@wallet-reducers/coinjoinReducer';
 
 import { AnonymityLevelSlider } from './AnonymityLevelSlider';
-import { DROPDOWN_MENU } from '@trezor/components/src/config/animations';
 
-const Container = styled.div`
+const SetupCard = styled(Card)`
     position: relative;
+    margin-bottom: 20px;
+    overflow: hidden;
 `;
 
-const SetupCard = styled(Card)`
+const Level = styled.div`
     position: absolute;
-    top: 12px;
-    right: 0;
-    width: 460px;
-    z-index: ${variables.Z_INDEX.BASE};
-    box-shadow: 0px 4px 4px ${({ theme }) => theme.BOX_SHADOW_BLACK_15};
-    border-radius: 16px;
-    animation: ${DROPDOWN_MENU} 0.15s ease-in-out;
+    right: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 64px;
+    height: 42px;
+    border: 1.5px solid ${({ theme }) => theme.STROKE_GREY};
+    border-radius: 8px;
+    color: ${({ theme }) => theme.TYPE_GREEN};
```

Comment:
I guess the color should be dynamic like this deleted util https://github.com/trezor/trezor-suite/pull/7107/files#diff-27b71a3f3ca869bef0e96dc5a4cbb3c429caf8debb20e84449e2ad637617577bL45 but it's ok to do it as part of the next PR with the bigger redesign.

Reply:
Yep, I left it out because I did minimal changes to logic and I wansn't sure we want to show that something is red and not good enough, will discuss it with Hynek later